core/templates/dev/head/components/forms/RequireIsFloatDirective.js

```diff
@@ -16,9 +16,7 @@
  * @fileoverview Directives for schema-based form builders.
```

Comment:
Please verify all fileoverviews in all files you have touched; attention to detail is important. This one is wrong.

Reply:
Verified all the file overviews -- sorry two file overviews were wrong